### PR TITLE
Add support for text dragging

### PR DESCRIPTION
**Pull request by marianacapelo**

### CODE REVIEW

File: WebViewControl/WebView.CefDragHandler.cs [hunk 1]
```diff
@@ -22,11 +22,19 @@ private class CefDragHandler : IDragHandler {
                     }
                 }
 
+                var textDragging = OwnerWebView.TextDragging;
```
Comment (joaompneves) [inline WebViewControl/WebView.CefDragHandler.cs:25]: why is this needed? I remember that that we had to do this for files because there was no way to get the filenames... but text could be obtain via js or is there any limitation?

Comment (marianacapelo) [reply WebViewControl/WebView.CefDragHandler.cs:25]: right. Text selection being dragged can be obtain via js only ondrop - which means we would only have knowledge of the text content being dragged when dropping in the target.
With this we allow the handling of such drags and its potential use (to set the object being dragged asap, for instance)

Comment (joaompneves) [reply WebViewControl/WebView.CefDragHandler.cs:25]: Ok I get it.

**Comment by joaompneves** [comment]
You need to raise the version of the webview and reactview... check their csprojs... there you will find their current versions... raise last digit

**Comment by marianacapelo** [comment]
> You need to raise the version of the webview and reactview... check their csprojs... there you will find their current versions... raise last digit

@joaompneves do you mean nuspec?
